src/patternfly/components/Login/examples/login-invalid-example.hbs

```diff
@@ -12,11 +12,9 @@
         {{#> login-main-header-desc}}
           Lorem ipsum dolor sit amet, consectetur adipiscing elit.
         {{/login-main-header-desc}}
-        {{#> dropdown id="invalid-login-demo-dropdown-select" dropdown--IsSelect="true" dropdown--ItemIsSelected="true" dropdown--HasToggleIcon="true"}}
-          {{#> dropdown-toggle-text}}
-            English
-          {{/dropdown-toggle-text}}
-        {{/dropdown}}
+        {{#> select id="login-invalid-select-single" select--IsLanguageMenu="true"}}
```

Comment:
The select is taking up 100% width on mobile due to the change we made in Core. I wonder if it should be wrapped in a container here.
<img width="541" alt="Screen Shot 2019-03-22 at 1 01 59 PM" src="https://user-images.githubusercontent.com/20118816/54840160-ba912c80-4ca2-11e9-9c8b-3b0b17704100.png">

Reply:
This is what I'm seeing on Sketch
<img width="541" alt="Screen Shot 2019-03-22 at 1 01 59 PM" src="https://user-images.githubusercontent.com/20118816/54840325-1f4c8700-4ca3-11e9-9c54-62bfb92b119b.png">

Reply:
<img width="217" alt="Screen Shot 2019-03-22 at 1 04 42 PM" src="https://user-images.githubusercontent.com/20118816/54845697-74db6080-4cb0-11e9-807f-dfeca8c72b2a.png">

Reply:
@gdoyle1 @mcarrano can you advise here? Since these designs were created, we have made the width of the select component 100%, so it fills the width of it's parent. We don't have a variation of the select component that will shrink back down to what it was before, where the toggle is as wide as what's in it, and the menu is as wide as the widest thing inside of it.

Just want to verify that we want selects to be 100% width by default? That's inconsistent with the other types of menus (dropdown, options menu, etc). Not saying it should be consistent, just some feedback.

And assuming so, should this select be full-width on mobile as seen in the first screenshot @christiemolloy provided? Or should we limit the width of the select? 

If we limit the width, should that be a variation we offer in the select component itself (for example, to offer a modifier that displays a select's toggle/menu width like a dropdown?)? Or should I use the layout in the login component to make the select some percentage or whatever of the available space?

Reply:
@mcoker I would not expect the Select to always fill it's container at 100%. I do remember some prior conversation on this where we said that the width or the toggle should not adjust based on the length of the selected item. But I didn't realize this meant that we were always sizing to 100%. I think I expected that you could set a fixed width on a Select list.

How do we deal with this in a toolbar, for example? We don't stretch the contents of the toolbar to fill the width of the screen, right?